statistics.py

```diff
@@ -1,5 +1,18 @@
 from load_chat import Chat, User
-from typing import Dict
+from typing import Dict, Callable
+
+
+def statistic(name: str) -> Callable[[Callable[[Chat], None]], Callable[[Chat], None]]:
+    def decorator(function: Callable[[Chat], None]) -> Callable[[Chat], None]:
+        def wrapper(chat: Chat) -> None:
+            number_of_eq = (80 - len(name) - 4) // 2
+            print(("=" * number_of_eq + "  " + name.upper() + "  " + number_of_eq * "=").center(80))
```

Comment:
В чем смысл всего этого кода? Почему нельзя просто
```suggestion
            print(f" {name.upper()} ".center(80, "="))
```